platforms/android/build_sdk.py

```diff
@@ -18,7 +18,7 @@ def execute(cmd, shell=False):
         log.info('Executing: ' + ' '.join(cmd))
         retcode = subprocess.call(cmd, shell=shell)
         if retcode < 0:
-            raise Fail("Child was terminated by signal:" %s -retcode)
```

Comment:
I think we need to keep a `-` sign here to print a positive error code.

Reply:
Change made.  I stuck with __%s__ to be consistent with the other error messages just below these lines.